packages/transaction/src/memo.ts

```diff
@@ -16,6 +16,22 @@ export const V2_MEMO_DEFAULT_ABI = Bytes4.from(
   }),
 )
 
+export const V2_MEMO_WITHDRAW_SIG_ABI = Bytes4.from(
+  abiCoder.encodeFunctionSignature({
+    name: 'decodeWithdrawSigMemoV2',
+    type: 'function',
+    inputs: [
+      {
+        type: 'bytes[]',
+        name: 'memo',
+      },
+    ],
+  }),
+)
+
+export const V2_MEMO_DEFAULT_ABI_ZERO = Bytes4.from('0x00000000')
```

Comment:
Is it to reduce the calldata gas cost using 00 instead of non-zero bytes that we use`V2_MEMO_DEFAULT_ABI_ZERO` and `V2_MEMO_WITHDRAW_SIG_ABI_ZERO`?

Reply:
Yes, we do a simple equality check for the hash. There is no need to use the keccak of a function signature, we don't use the abicoder elsewhere.

I left the old values in for backward compatibility with our current testnet.